spec/models/user_spec.rb

```diff
@@ -1226,4 +1226,27 @@
       end
     end
   end
+
+  describe '#onboarding' do
+    let(:user) { create(:user) }
+
+    before do
+      Flipper.enable(:veteran_onboarding_beta_flow, user)
+      create(:user_verification, idme_uuid: user.idme_uuid)
+    end
+
+    it 'show_onboarding_flow_on_login returns true when flag is enabled and display_onboarding_flow is true' do
```

Comment:
you'll want a context that describes when the underlying 'Veteran Onboarding' model is false vs true